### PR TITLE
shared workflow for our copilot deploys

### PR DESCRIPTION
**Pull request by codekitchen**

### CODE REVIEW

File: .github/workflows/copilot-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+          role-to-assume: ${{ inputs.role_to_assume }}
+          role-duration-seconds: 3600
+          role-session-name: GithubActionDeploy
+
+      - name: Setup AWS Copilot
+        uses: flemzord/setup-aws-copilot@v1.3
```
Comment (erikols) [inline .github/workflows/copilot-deploy.yml:56]: Probably ignorance on my part, but wondering if this is a stable base for this. If I search for e.g. "flemzord/setup-aws-copilot", I seem to just get gitmemory.cn refs. Don't let this stop you from merging, I just wanted to ask where this comes from.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
 # .github
-GitHub metadata and shared workflows
+
+GitHub metadata and shared workflows. The repo name `.github` has special significance to GitHub within an organization, so don't treat this as a normal code repository.
+
+## Shared Workflows
+
+GitHub requires shared workflows to be in a public repo, we are storing them in here. These workflows are specific to Lumio and aren't believed to be generally useful outside our organziation, but they are MIT licensed.
```
Comment (erikols) [inline README.md:7]: I guess unless it's enterprise, but effectively for now, it does

File: .github/workflows/copilot-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ inputs.git_ref }}
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
```
Comment (erikols) [inline .github/workflows/copilot-deploy.yml]: Is there any reason to tokenize the region for now? No worries if not. just wondered if the var on line 23 was meant to expand here

**Review by erikols — APPROVED** [review]
I'm new to this, but it looks good to me as is, along with the policy page

File: .github/workflows/copilot-deploy.yml (hunk 1 above)
Comment (codekitchen) [reply .github/workflows/copilot-deploy.yml:56]: We found this on the marketplace at https://github.com/marketplace/actions/setup-aws-copilot-cli

That is a valid point though, perhaps this is simple enough that we should do it ourselves

File: README.md (hunk 2 above)
Comment (codekitchen) [reply README.md:7]: self-hosted enterprise yeah (not enterprise cloud)

File: .github/workflows/copilot-deploy.yml (hunk 3 above)
Comment (codekitchen) [reply .github/workflows/copilot-deploy.yml]: nope, good catch thanks. will update

**Pull request merged**